sdk-api-src/content/accountssettingspaneinterop/index.md

```diff
@@ -15,6 +15,7 @@ f1_keywords:
 
 ## -description
 
-This header is used for WinRT Interoperability. For more information, see:
+This header is used by Windows Runtime C++ reference. For more information, see:
+
+- [Windows Runtime C++ reference](../_winrt/index.md)
```

Comment:
There's a couple WinRT headers that used to link to a more specific file (`/uwp/api/Windows.UI.ApplicationSettings.AccountsSettingsPane` in this case).

That's, however, inconsistent with how all the other index docs link back to their tech tree. Perhaps the more direct link should be under a `See Also` section or something similar and with a different paragraph/link style so tooling can differentiate?

Reply:
@tannergooding For this one, the text calling out "WinRT Interoperability" was chosen deliberately to call out the specific developer scenario for these APIs. It's true that it's inconsistent with the other headers, but this was specifically requested by the PMs for the WinUI platform. Can you revert this change in just the *interop headers?

Reply:
Can do.